packages/@aws-cdk/aws-lambda-event-sources/test/test.dynamo.ts

```diff
@@ -231,4 +231,46 @@ export = {
     test.done();
   },
 
+  'contains eventSourceMappingId after lambda binding'(test: Test) {
+    // GIVEN
+    const stack = new cdk.Stack();
+    const fn = new TestFunction(stack, 'Fn');
+    const table = new dynamodb.Table(stack, 'T', {
+      partitionKey: {
+        name: 'id',
+        type: dynamodb.AttributeType.STRING
+      },
+      stream: dynamodb.StreamViewType.NEW_IMAGE
+    });
+    const eventSource = new sources.DynamoEventSource(table, {
+      startingPosition: lambda.StartingPosition.TRIM_HORIZON
+    });
+
+    // WHEN
+    fn.addEventSource(eventSource);
+
+    // THEN
+    test.notEqual(eventSource.eventSourceMappingId, undefined);
```

Comment:
Use `test.ok()`. 

I would even suggest that you test for equality to the event source id that gets returned here - it'll be a cloudformation intrinsic of the form 'Fn::GetAtt: { ... }'.

In case, it starts with 'Token...', use `stack.resolve(eventSource.eventSourceMappingId)` to resolve the token and get the CF intrinsic.

Reply:
I changed it to test.ok but could you maybe elaborate on how I should check equality of the eventSourceId? 

`stack.resolve(eventSource.eventSourceMappingId)` returns `{Ref: '...'}` where shall I get the id that should be equal to this Ref value?

Reply:
You cannot. The eventual evenSourceMappingId is generated by cloudformation and our unit tests don't interact with it. You can only check for direct equality with the `Ref` string.